rest.md

```diff
@@ -137,8 +161,7 @@ I'm going to search for "toast". This takes me to a new URL:
 
 https://twitter.com/search?q=toast&src=typd
```

Comment:
Maybe wrap these in backticks so they stand out from the normal font in the lesson plan.